### DISCUSSION TITLE
Welcome to irc7 Discussions!

### DISCUSSION THREAD

**Discussion by jyonxo** [irc7-com/irc7#38] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by n0stalzha** [comment]
Hi there,

I'm planning my own web chat service as a learning project using C# .NET Core, SignalR and MVC.

Creating an IRC client and server is also something I had in mind...

I'd also like to practice some collaboration, and learn from/contribute to a project so I thought that I'd introduce myself.

There are some questions I have, but would prefer to discuss over email or chat. I got to this repo though a msnld wiki which seems pretty derelict. I tried connecting to mschat.msnld.com but that seems to no longer be online...

**Comment by jyonxo** [comment]
Hi @n0stalzha ,

Thank you for your interest, we always welcome anyone who is able to contribute in anyway to the project. At the moment the codebase is pretty rough unfortunately I have not had much time so parts are hacked in and need refactoring. We do have a discord with an #irc7 channel so if you would like to join you are most welcome. 

The server itself covers a few flavors of IRC, the 1459 draft and associated drafts, the IRCX draft and finally the MSN Chat protocol which is not well documented. We can discuss the project there, hopefully give some meaningful insight that may help you in your project. 

The discord invite is here:
https://discord.gg/EMFDDvC2

The project is found under a bigger project named Mono.chat where we will also eventually have a webchat there as well:
https://www.mono.chat/

The namespace (group name) will also change to reflect this, and MSNLD will be dropped. I look forward to chatting with you on the discord!
